profile_report.py

```diff
@@ -0,0 +1,31 @@
+from profiler import data
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("benchmark", type=str)
+parser.add_argument("path", type=str)
+
+args = parser.parse_args()
+
+# Processes the set of previously written logs
+
+threshold = 1.10  # Percent difference
+
+db = data.FileBasedProfileDB(args.path)
+actual_max_ts = 0
+dt = db.find(f"{args.benchmark}")
+c = 0
+
+if len(dt) >= 2:
+    exec_time = [0.0, 1.0]
+    last_two = dt[-2:]
+    exec_time[0] = last_two[0].user_time_sec
+    exec_time[1] = last_two[1].user_time_sec
```

Comment:
more idiomatic as:
```suggestion
exec_times = dt[-2].user_time_sec, dt[-1].user_time_sec
```
(note the pluralization, which would require edits below)